docs/03-Usage.md

```diff
@@ -168,11 +167,12 @@ The [compatibility module](11-Submodules/01-Fulltext-Search-Compatibility.md) ha
 
 ## Grouped indexing
 
-Note, grouped indexing is `0` based. Thus, if there are 150 groups to index, the final group to index will be 149 instead of 150.
+Grouped indexing is `0` based. Thus, if there are 150 groups to index, the final group to index will be 149 instead of 150.
```

Comment:
Maybe "Be aware that grouped indexing is `0` based"? Just to make it a bit more obvious?